plugins/leemons-plugin-tests/frontend/src/components/QuestionImage.js

```diff
@@ -34,29 +35,30 @@ export function QuestionImage({ src, markers, values, clue }) {
       <LeebraryImage className={classes.image} src={src} />
       {markers && markers.list
         ? markers.list.map((marker, index) => {
```

Comment:
If you see in detail, there must be 2 identation levels:

```js
markers && markers.list
	? markerts.list.map((marker, index) => {
			// ...
		})
	: null;
```

Reply:
fixed